dyn/tm/accounts.py

```diff
@@ -408,6 +409,18 @@ def _post(self, password, email, first_name, last_name, nickname,
         """Create a new :class:`~dyn.tm.accounts.User` object on the DynECT
         System
         """
+
+        api_args = {'password': password, 'email': email,
```

Comment:
`permissions` needs to be `permission` and (sadly/tediously) all these values? Yep, they're going to have to be added conditionally. That is, if they are set, send them, if not, don't.